src/Microsoft.Crank.Jobs.HttpClient/Program.cs

```diff
@@ -477,28 +514,41 @@ public static async Task<WorkerResult> DoWorkAsync()
             {
                 var requestMessage = new HttpRequestMessage
                 {
-                    Method = new HttpMethod(timeline.Method),
+                    Method = timeline.Method,
                     Content = timeline.HttpContent
                 };
 
+                if (Body != null)
+                {
+                    requestMessage.Content = new ByteArrayContent(Body);
+                }
+
                 foreach (var header in timeline.Headers)
                 {
-                    requestMessage.Headers.TryAddWithoutValidation(header.Key, header.Value);
+                    if (!requestMessage.Headers.TryAddWithoutValidation(header.Key, header.Value.TrimStart()))
+                    {
+                        requestMessage.Content.Headers.TryAddWithoutValidation(header.Key, header.Value.TrimStart());
```

Comment:
If there is no body then `Content` is null and this will error.

Reply:
Thanks, will provide a PR to fix it